### PR TITLE
Add LICENSE

### PR DESCRIPTION
**Pull request by BoPeng**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) Bo Peng and the University of Texas MD Anderson Cancer Center
```
Comment (hmaarrfk) [inline LICENSE]: the license ownership seems strange no?

Comment (BoPeng) [reply LICENSE]: Sorry, I simply copied my LICENSE file without looking. Will revise.

Comment (BoPeng) [reply LICENSE]: Fixed. Good try to steal other's work though. :-)

**Comment by blink1073** [comment]
We will need a Manifest.in file that includes the
license

File: LICENSE (hunk 1 above)
Comment (hmaarrfk) [reply LICENSE]: But somebody needs to claim copyright ;)

Comment (blink1073) [reply LICENSE]: I think "calysto developers" is the right name

Comment (BoPeng) [reply LICENSE]: https://github.com/Calysto/metakernel/blob/master/LICENSE.txt uses 

```
Copyright (c) 2014-2019, IPython Kernel Development Team and Metakernel Development Team
```

Comment (blink1073) [reply LICENSE]: That one did use some code from IPython.  I think Calysto is more appropriate here.

Comment (BoPeng) [reply LICENSE]: @blink1073 I have added a license line but it can be easier for you to add the file with proper term by yourself...

**Comment by BoPeng** [comment]
added

File: LICENSE (hunk 1 above)
Comment (blink1073) [reply LICENSE]: I am afk for the night, "Calysto Development Team" is the term I would use

**Comment by hmaarrfk** [comment]
If this does add things to the SDIST, then I will be damned,

When I was trying, this is what I needed
https://github.com/scikit-image/scikit-image/blob/master/setup.cfg

**Comment by BoPeng** [comment]
I can confirm that with `MANIFEST.in`, the `sdist` package contains `LICENSE` file.

**Comment by blink1073** [comment]
Thanks!

**Comment by blink1073** [comment]
I'll cut a release in the morning

**Comment by blink1073** [comment]
All set!

**Pull request merged**